# message to represent control of the various torso servos. Values are floats
# -1 to 1, where -1 is the minimum extreme, 1 is the maximum extreme, and 0 is
# therefore 0.
float32 chest
float32 left_elbow
float32 right_elbow
float32 head
float32 tail

bool lock_wheels

float32 DOWN
float32 UP
